src/audio/module_adapter/module/waves.c

```diff
@@ -343,16 +343,13 @@ static int waves_effect_init(struct processing_module *mod)
 	}
 
 	waves_codec->request_max_bytes = 0;
-	waves_codec->response_max_bytes = 0;
-	waves_codec->response = 0;
```

Comment:
to the commit message - it the whole message in the subject? Please make a short (<70 characters, actually I think even shorter is recommended) subject and move the rest to the commit message body